### ISSUE TITLE
Error in creating ref sample

### ISSUE THREAD

**Issue by toma5692** [burtonrj/CytoPy#24]
Hi Ross, hope you are doing well. 

Small issue in creating ref samples in your tutorial 03. 

When I run : create_ref_sample(experiment=cells, sample_size=5000, root_population="root",
                  new_file_name="Training Data")

I get    
100     features = [x for x in data.columns if x != "sample_id"]
--> 101     new_filegroup = FileGroup(primary_id=new_file_name)
    102     new_filegroup.data_directory = experiment.data_directory
    103     new_filegroup.init_new_file(data=data[features].values,

/usr/local/lib/python3.8/dist-packages/cytopy/data/fcs.py in __init__(self, *args, **kwargs)
    218         else:
    219             if any([x is None for x in [data, channels, markers]]):
--> 220                 raise ValueError("New instance of FileGroup requires that data, channels, and markers "
    221                                  "be provided to the constructor")
    222             self.save()

ValueError: New instance of FileGroup requires that data, channels, and markers be provided to the constructor

You may have changed the way you instantiate a filegroup, right ? 

Thanks, 

Thomas

**Comment by burtonrj** [comment]
Hi Thomas,

Yeah this is an odd error, no idea how I didn't catch this in my tests. There are a couple of bugs I'm fixing at the moment with a new release coming on Monday, so I'll make sure to include a fix for this issue in the new release.

For the time being you can create a reference sample manually by running the following:

```python
from cytopy.flow.variance import load_and_sample
# Sample all of the FileGroups from your Experiment
data = load_and_sample(experiment=cells,
                                         population='root',
                                         sample_size=5000,
                                         sample_ids=cells.list_samples(),
                                         sampling_method='uniform',
                                         transform=None)[0]
channels = [x for x in data.columns if x != 'sample_id']
new_filegroup = FileGroup(primary_id='Training_Data', channels=channels, markers=channels)
new_filegroup.notes = 'sampled data'
new_filegroup.save()
cells.fcs_files.append(new_filegroup)
cells.save()
```

**Comment by toma5692** [comment]
Thanks !!

**Comment by burtonrj** [comment]
Issue addressed in latest release v2.0.1 - new docker image pushed and will be online in the next few hours
